server/vault.api.js

```diff
@@ -55,6 +63,7 @@ const findBrews = async (req, res)=>{
 
 	await HomebrewModel.find(combinedQuery, projection)
 		.skip(skip)
+		.sort(sortConditions(sort, dir))
```

Comment:
Sorting must be performed before any `limit` or `skip` step. Otherwise it is only sorting on the current page, and later pages will have results that should have gone before.

Reply:
Does not

Reply:
> When you chain [skip()](https://www.mongodb.com/docs/manual/reference/method/cursor.skip/#mongodb-method-cursor.skip) and [limit()](https://www.mongodb.com/docs/manual/reference/method/cursor.limit/#mongodb-method-cursor.limit), the method chaining order does not affect the results. The server always applies the skip operation based on the sort order before it applies the limit on how many documents to return.

Best i could find in the official docs to support that statement.